src/transformers/models/sew_d/configuration_sew_d.py

```diff
@@ -63,8 +63,6 @@ class SEWDConfig(PretrainedConfig):
             Whether to share attention key with c2p and p2c.
         relative_attention (`bool`, *optional*, defaults to `True`):
             Whether to use relative position encoding.
-        position_biased_input (`bool`, *optional*, defaults to `False`):
```

Comment:
This is in the checkpoint conversion script, but never used anywhere else.

It appears for `deberta` and `deberta_v2` where this attribute are actually used